CHANGELOG.md

```diff
@@ -0,0 +1,63 @@
+- fix radiance conversion for mode 4+. The reference files were not being binned etc, this is now now done during tfm_setup. This also removes redundant work in the dn2rad function. The current structure does not work for mode 5 this way, which needs to be re thought (due to wavelength def being smaller then full array after smile correction).
+
+
+
+## 0.1.0
+
+### New Features
+
+- More Updates with hardware, updated automated calibration, other tweaks. ([#13](https://github.com/openhsi/openhsi/pull/13)), thanks to [@christhechris](https://github.com/christhechris)
+  - - many updates to cal code for robustness.
+- tweaks to crop settings
+- add more flags tfm setup
+- add some plotting functions to plot raw data
+- update cal file with new property for cal luminance, previously hard coded.
+- fix nbs path issues from swap to notebooks in nbs folder.
+- fix radiance conversion for mode 4+. The reference files were not being binned etc, this is now now done during tfm_setup. This also removes redundant work in the dn2rad function. The current structure does not work for mode 5 this way, which needs to be re thought (due to wavelength def being smaller then full array after smile correction).
+
+
+
+## 0.1.0
+
+### New Features
+
+- More Updates with hardware, updated automated calibration, other tweaks. ([#13](https://github.com/openhsi/openhsi/pull/13)), thanks to [@christhechris](https://github.com/christhechris)
+  - - many updates to cal code for robustness.
+- tweaks to crop settings
+- add more flags tfm setup
+- add some plotting functions to plot raw data
+- update cal file with new property for cal luminance, previously hard coded.
+- fix nbs path issues from swap to notebooks in nbs folder.
+- fix radiance conversion for mode 4+. The reference files were not being binned etc, this is now now done during tfm_setup. This also removes redundant work in the dn2rad function. The current structure does not work for mode 5 this way, which needs to be re thought (due to wavelength def being smaller then full array after smile correction).
+
+
+
+## 0.1.0
+
+### New Features
+
+- More Updates with hardware, updated automated calibration, other tweaks. ([#13](https://github.com/openhsi/openhsi/pull/13)), thanks to [@christhechris](https://github.com/christhechris)
+  - - many updates to cal code for robustness.
+- tweaks to crop settings
+- add more flags tfm setup
+- add some plotting functions to plot raw data
+- update cal file with new property for cal luminance, previously hard coded.
+- fix nbs path issues from swap to notebooks in nbs folder.
+- fix radiance conversion for mode 4+. The reference files were not being binned etc, this is now now done during tfm_setup. This also removes redundant work in the dn2rad function. The current structure does not work for mode 5 this way, which needs to be re thought (due to wavelength def being smaller then full array after smile correction).
```

Comment:
I didn't realise we had a changelog. good idea. Looks like there are repeats though.